features-support/step_definitions/features.js

```diff
@@ -24,11 +24,27 @@ function getFeaturesFromUrl(callback) {
     });
 }
 
+var staticTestDataExists = false;
+
 module.exports = function () {
+
+  // Create static test data, only do this once.
   this.Given(/^a set of specifications containing at least one feature file\.?$/, function (callback) {
     var world = this;
-    world.createSpecsForTesting()
+
+    // Only generate the static test data once.
+    if (staticTestDataExists) {
```

Comment:
Does this not duplicate what cleanSlate tag is doing? Could we not just tag a feature file with @cleanSlate?